apps/service_broadcast/lib/broadcast/init.ex

```diff
@@ -5,6 +5,8 @@ defmodule Broadcast.Init do
 
   def on_start(state) do
     Broadcast.Stream.Store.get_all!()
+    |> Enum.reject(&is_nil/1)
```

Comment:
why would we be getting `nil`s back? is this a side effect of confusion when we're querying expecting an atom key and all maps in the view state are going to have string keys?

Reply:
I believe so. It could also come from deleting the inner map (event/struct) but leaving transformations or something else laying around. In the end it seemed better to be defensive and save ourselves from a future-footgun.

Reply:
feels like if we're persisting `nil`s into an app's view state that _is_ a footgun

Reply:
That's orthogonal to whether we mark done or do another collection. We should NOT be persisting nils, but the code was already persisting nils when we grabbed this card, due to the atoms / strings thing. We're rejecting nils here so that if other code is written like the atom keys thing later, we're not back into a broken state.